code/modules/projectiles/guns/misc/hygienegun.dm

```diff
@@ -0,0 +1,57 @@
+/obj/item/gun/hygienegun
```

Comment:
pacifist check? also why not make it use soap as mags

Reply:
I'll consider making it use soap as ammo